Add cluster-port parameter (#1664)
redis/redis-doc: topics/cluster-tutorial.md

---

Every Redis Cluster node requires two TCP connections open. The normal Redis
TCP port used to serve clients, for example 6379, plus the second port named 

> cluster bus port. The cluster bus port could be specified in redis.conf file, or it could
> be obtained by adding 10000 to the data port, so 16379 in the example.

Reviewer: ```suggestion
cluster bus port. The cluster bus port will be derived by adding 10000 to the data port, 16379 in this example, or by overiding it with the cluster-port config.
```
